packages/simcore-sdk/src/simcore_sdk/node_ports_v2/port_utils.py

```diff
@@ -67,7 +67,7 @@ async def get_value_from_link(
         if file_to_key_map:
             file_name = next(iter(file_to_key_map))
 
-        file_path = data_items_utils.create_file_path(key, file_name)
+        file_path = data_items_utils.get_file_path(key, file_name)
```

Comment:
I do not understand this change. how is this a get now??
it is still creating a file path. this has nothing to do with an effective file

Reply:
I really do not agree with it. This naming created a lot of confusion. It does not create anything. It returns a path that does not exist. For example PC thought it would return you the path to an existing directory. But that is not True. I've decided to use a more accurate name.